portfolio/src/main/java/com/google/sps/servlets/LikeDataServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+/** Servlet responsible for liking comments. */
+@WebServlet("/like-data")
+public class LikeDataServlet extends HttpServlet {
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long likes = Long.parseLong(request.getParameter("likes"));
+    long id = Long.parseLong(request.getParameter("id"));
+    likes ++;
+  
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment");
```

Comment:
Avoid hard-coded strings here and elsewhere.

Given that some of these are shared between multiple .java files, one approach you can take to avoid duplication is to create a new file, say `Constants.java`.

Then, define these as public static constants, e.g.

```java
public static final String SOME_PARAM_NAME = ...;
```

And then use then in each of your Servlets